src/main/java/com/bunq/sdk/model/generated/Card.java

```diff
@@ -162,6 +162,14 @@
   @SerializedName("pin_code_assignment")
   private List<CardPinAssignment> pinCodeAssignment;
 
+  /**
+   * ID of the MA to be used as fallback for this card if insufficient balance. Fallback account
```

Comment:
Shouldn't  `MAD be spelled out completely ?

Reply:
`ma` *

Reply:
generated